builtin/verify-tag.c

```diff
@@ -51,8 +56,20 @@ int cmd_verify_tag(int argc, const char **argv, const char *prefix)
 		const char *name = argv[i++];
 		if (get_sha1(name, sha1))
 			had_error = !!error("tag '%s' not found.", name);
-		else if (gpg_verify_tag(sha1, name, flags))
-			had_error = 1;
+		else {
+			if (fmt_pretty) {
+				verify_ref_format(fmt_pretty);
```

Comment:
Yeah, that's a good alternative. Although it may not be necessary if we merge this code with git-verify-tag